javascript/javascript1/week2/weather-wear.js

```diff
@@ -0,0 +1,9 @@
+function weatherWear(temperature) {
```

Comment:
Nice... But it would be really good if you had more conditions and also an else statement at the end.